massa-proto/proto/massa/api.proto

```diff
@@ -0,0 +1,302 @@
+syntax = "proto3";
+
+package massa.api.v1;
+
+import "common.proto";
+import "slot.proto";
+
+import "google/api/annotations.proto";
+import "google/rpc/status.proto";
+import "block.proto";
+import "operation.proto";
+
+option cc_enable_arenas = true;
+option csharp_namespace = "Massa.Api.V1";
+option go_package = "google.golang.org/genproto/massa/api/v1/grpc;grpc";
+option java_multiple_files = true;
+option java_package = "com.massa.api.v1";
+option objc_class_prefix = "GRPC";
+option php_namespace = "Massa\\Api\\V1";
+
+// Massa gRPC service
+service Grpc {
+    // GetDatastoreEntries
+    rpc GetDatastoreEntries (GetDatastoreEntriesRequest) returns (GetDatastoreEntriesResponse) {
+        option (google.api.http) = {
+            post: "/v1/get_datastore_entries"
+            body: "*"
+        };
+    }
+
+    // GetSelectorDraws
+    rpc GetSelectorDraws (GetSelectorDrawsRequest) returns (GetSelectorDrawsResponse) {
+        option (google.api.http) = {
+            post: "/v1/get_selector_draws"
+            body: "*"
+        };
+    }
+
+    // GetVersion
+    rpc GetVersion (GetVersionRequest) returns (GetVersionResponse) {
+        option (google.api.http) = {get: "/v1/version"};
```

Comment:
We should adopt a convention: `/v1/NAME_OF_THE_METHOD`everywhere.  As we couldn't really split in `resources`

this is valid in similare cases